source/change-streams/tests/change-streams.yml

```diff
@@ -456,3 +456,77 @@ tests:
             coll: *collection_name
         -
           operationType: invalidate
+  -
+    description: Test consecutive resume
+    minServerVersion: "3.6.0"
```

Comment:
I think we should make this 4.1.7 to prevent secondaries from triggering the fail point: https://jira.mongodb.org/browse/SERVER-34943

Reply:
Done.  I'll put up a patch release of the Java driver before merging this.  It's passing the test locally.

Reply:
Java driver tests are passing